doc/user/content/get-started.md

```diff
@@ -73,51 +54,64 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
 {{< /tab >}}
 {{< /tabs >}}
 
-### Prepare your environment
+## Step 1. Start a free trial
 
-Before getting started, let's prepare an isolated environment for experimenting that guarantees we won't interfere with other workloads running in your database. As a first step, create a new [cluster](/overview/key-concepts/#clusters) with dedicated physical resources:
+With a free trial, you get 14 days of access to Materialize resources worth up to [4 credits per hour](https://materialize.com/pricing/). This limit should accommodate most trial scenarios. However, if you need additional time or resources, reach out to sales@materialize.com.      
 
-```sql
-CREATE CLUSTER quickstart REPLICAS (small_replica (SIZE = '2xsmall'));
-```
+1. [Sign up for a Materialize trial account](https://materialize.com/register/) using your business email address. 
 
-And make it our active cluster:
+1. Check your inbox for a confirmation email from Materialize and follow the instructions to activate your account.
 
-```sql
-SET cluster = quickstart;
-```
+    In most cases, your confirmation should arrive in 30 minutes or less.
 
-Now that **physical isolation** is out of the way, let's also take care of **logical isolation** and create a new schema to make sure we don't run into [namespacing](/sql/namespaces/) annoyances (like naming collisions):
+## Step 2. Prepare your environment
 
-```sql
-CREATE SCHEMA qck;
```

Comment:
Schemas are used for a selfish purpose in the current get started: to differentiate in our analytics which objects were created as part of the demo from others. There's no reason a user has work around this for us, but we should consider some other object naming conventions (like a prefix) we can reliably assume are "ours" and not a user's organic ones

Reply:
For now, going with a `_qck` suffix.